packages/venia-ui/lib/components/CommunicationsPage/communicationsPage.css

```diff
@@ -0,0 +1,50 @@
+.root {
+    padding: 20px;
+    font-size: 0.875rem;
+}
+
+.header {
+    padding-bottom: 10px;
+}
+
+.title {
+    margin-bottom: 25px;
+    text-align: center;
+    font-weight: 700;
```

Comment:
Checkout `packages/venia-ui/lib/tokens.css`, which will hold tokens to keep style consistent.